packages/sdk-cli/src/models/localRelay/launch.test.ts

```diff
@@ -3,6 +3,7 @@ import { ChildProcess } from 'child_process';
 import { join } from 'path';
 
 import { launch } from './launch';
+import { console } from 'fp-ts';
```

Comment:
This seems like an accidental auto-import? I think you want console, the Node global?